lib/librato/rack/configuration.rb

```diff
@@ -80,12 +83,28 @@ def source=(src)
 
       def dump
         fields = {}
-        %w{user token log_level source prefix flush_interval source_pids}.each do |field|
+        %w{user token log_level source prefix flush_interval source_pids suites}.each do |field|
           fields[field.to_sym] = self.send(field)
         end
+        fields[:metric_suites] = metric_suites.fields
         fields
       end
 
+      def metric_suites
+        @metric_suites ||= case suites.downcase.strip
+                           when 'all'
+                             SuitesAll.new
+                           when 'none'
+                             SuitesNone.new
+                           when /^\+/
+                             SuitesInclude.new(suites[1..-1])
+                           when /^\-/
+                             SuitesExcept.new(suites[1..-1])
```

Comment:
I think something like `LIBRATO_SUITES=+memory,-rack_status` is not an unlikely case in the future but we can always cross that bridge in a later PR.